README.md

```diff
@@ -135,6 +135,52 @@ The cluster creation and retrieving connection info can be done directly in comm
   hzc --cluster-name <CLUSTER NAME> --cloud-token <DISCOVERY TOKEN>
   ```
 
+## SSL Configuration
+
+You can use the following configuration file to enable SSL support:
+```
+ssl:
```

Comment:
We should try all these scenarios to see if they work

Reply:
We tried them manually and they seem to work.